src/app/main/main.component.js

```diff
@@ -40,6 +49,21 @@ function routingConfig($stateProvider, $locationProvider, $urlRouterProvider){
       url: "/checkout.html",
       template: '<checkout></checkout>'
     })
+    .state('profile', {
+      url: "/profile",
+      abstract: true,
+      template: '<profile></profile>',
+      onEnter: (sessionService,sessionModalService,$state) => {
```

Comment:
The UI router is only used for the local dev env, it's not used once the app is integrated with the CMS.  So, you'll want to implement this using the session enforcement service @Omicron7 made.

Here is an example on the checkout page: https://github.com/CruGlobal/give-web/blob/master/src/app/checkout/checkout.component.js#L39